owa/app/components/admissionLocation/admissionLocationWrapper.js

```diff
@@ -31,34 +32,28 @@ export default class AdmissionLocationWrapper extends React.Component {
 
         this.intl = context.intl;
         this.fetchAllAdmissionLocations = this.fetchAllAdmissionLocations.bind(this);
+        this.fetchAllVisitLocations = this.fetchAllVisitLocations.bind(this);
         this.fetchBedTypes = this.fetchBedTypes.bind(this);
         this.getBody = this.getBody.bind(this);
         this.fetchAllAdmissionLocations(this);
+        this.fetchAllVisitLocations(this);
         this.fetchBedTypes();
     }
 
     fetchAllAdmissionLocations(self) {
         axios
             .get(this.urlHelper.apiBaseUrl() + '/location', {
                 params: {
+                    tags: 'Admission Location',
                     v: 'full'
                 }
             })
-            .then(function(response) {
-                const admissionLocationUuidList = [];
-                const visitLocationUuidList = [];
-                _.forEach(response.data.results, (location) => {
-                    _.each(location.tags, (tag) => {
-                        if (tag.display == 'Admission Location') {
-                            admissionLocationUuidList.push(location.uuid);
-                        } else if (tag.display == 'Visit Location') {
-                            visitLocationUuidList.push(location.uuid);
-                        }
-                    });
-                });
+            .then(function (admissionResponse) {
```

Comment:
"admissionResponse" seems like a misleading name. Would naming something in lines of "admissionLocationsResponse" be better?